pom.xml

```diff
@@ -228,6 +228,19 @@
                     <compatibleSinceVersion>2.2.0</compatibleSinceVersion>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <version>3.1.12</version>
+                <dependencies>
+                    <!-- overwrite dependency on spotbugs if you want to specify the version of spotbugs -->
+                    <dependency>
+                        <groupId>com.github.spotbugs</groupId>
+                        <artifactId>spotbugs</artifactId>
+                        <version>4.0.0-beta3</version>
```

Comment:
If this is to fix some false positive, I think it would be better to update [plugin-pom](https://github.com/jenkinsci/plugin-pom/blob/2b11d0c2656a95ad9dfefb179d6b7c6a614e9e78/pom.xml#L626) directly and suppress the broken warning here or update to the new version of the parent pom.

Reply:
this is just to run SpotBugs locally

Reply:
I think that should be possible already with `mvn spotbugs:check`? By default, the configuration in the parent pom only runs SpotBugs as part of the `verify` phase (i.e. `mvn install` or `mvn verify`), so maybe that was the confusion?